### ISSUE TITLE
Custom Layer - Background Map goes white when clearing using threejs renderer

### ISSUE THREAD

**Issue by gallickgunner** [mapbox/mapbox-gl-js#9499]
**mapbox-gl-js version**: (DeckGL 8.1.0 + react-map-gl 5.2.3)

Hello, I was experimenting with loading models using the custom layer interface and ran into a problem. I'm actually drawing the custom layer through DeckGL and react-map-gl, however I believe that the heart of the problem lies in the custom layer which belongs here so I'm asking the question here. Anyways the problem is pretty similar to issue #8936. However the solution there doesn't work but infact, it's the cause of the problem as far as I understood. If you try clearing manually or set `renderer.autoClear` to true, the underlying map disappears and only a white background is shown with the model laid on top. 

This can be easily seen using the official [example's](https://docs.mapbox.com/mapbox-gl-js/example/add-3d-model/) and it's [jsfiddle](https://jsfiddle.net/api/post/library/pure/). Just set `autoClear` to true.

I was actually experimenting with using post processing, which requires to use `composer.render` instead of `renderer.render` and the former clears the buffers during passes. 

I thought this was due to a non transparent canvas issue perhaps and initialized the threejs renderer with a separate canvas, with `alpha` set to true and setting the clear color appropriately but nothing happens.

Sorry but I couldn't find out what `mapbox-gl-js` version the current `react-map-gl` uses. Also I apologize in advance for not providing any other necessary information but I'm new to react/js/mapbox and web programming. Thanks.

**Comment by xawill** [comment]
Any updates on this ?

**Comment by gallickgunner** [comment]
> Any updates on this ?

Nope. I couldn't find the reason so left it and focused on differet things. This almost seems like the threejs renderer also tirggers a clear in the deckgl renderer or something along those lines.

**Comment by xawill** [comment]
@Mugen87 Are you able to give us some ideas on this or to point us to another three.js major contributor ?

**Comment by zhiqunDLUT** [comment]
i found same questions ,custom layer in three.js

and i use threebox there would be another question refresh error
![image](https://user-images.githubusercontent.com/31470793/148198662-f8fec4ea-ec24-4e2a-b99d-857a1ae98f21.png)
